pkg/syncer/cnsoperator/controller/cnsfileaccessconfig/cnsfileaccessconfig_controller.go

```diff
@@ -78,6 +78,21 @@ func Add(mgr manager.Manager, clusterFlavor cnstypes.CnsClusterFlavor,
 		log.Debug("Not initializing the CnsFileAccessConfig Controller as its a non-WCP CSI deployment")
 		return nil
 	}
+
+	if clusterFlavor == cnstypes.CnsClusterFlavorWorkload {
```

Comment:
Are we avoiding just enabling the controller or do we ignore registering the instance itself?

Reply:
only the controller will be disabled with this code. Let me stop registering CR as well.